documentation/Installation.md

```diff
@@ -20,7 +20,7 @@ Saffire. Once the box has started, issue the following:
 
     vagrant ssh
     cd /vagrant
-    sh build.sh
+    sudo ./build.sh -i
```

Comment:
I wonder, is sudo permission really needed to build? I assume yes for installing globally, but not sure if that works correctly yet.